src/renderers/shared/fiber/ReactFiberExpirationTime.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @providesModule ReactFiberExpirationTime
+ * @flow
+ */
+
+'use strict';
+
+import type {PriorityLevel} from 'ReactPriorityLevel';
+const {
+  NoWork,
+  SynchronousPriority,
+  TaskPriority,
+  HighPriority,
+  LowPriority,
+  OffscreenPriority,
+} = require('ReactPriorityLevel');
+
+const invariant = require('fbjs/lib/invariant');
+
+// TODO: Use an opaque type once ESLint et al support the syntax
+export type ExpirationTime = number;
+
+const Done = 0;
+const Sync = 1;
+const Task = 2;
+const Never = Number.MAX_SAFE_INTEGER;
+
+const UNIT_SIZE = 10;
+const MAGIC_NUMBER_OFFSET = 10;
+
+exports.Done = Done;
+exports.Never = Never;
+
+// 1 unit of expiration time represents 10ms.
+function msToExpirationTime(ms: number): ExpirationTime {
+  // Always add an offset so that we don't clash with the magic number for Done.
+  return Math.round(ms / UNIT_SIZE) + MAGIC_NUMBER_OFFSET;
```

Comment:
Why round? (Faster to do `(ms / UNIT_SIZE) | 0`.)

Reply:
`ms / UNIT_SIZE) | 0` equals `Math.floor` rather than `Math.round`, right?